src/libpriv/rpmostree-postprocess.c

```diff
@@ -787,6 +787,41 @@ replace_nsswitch (int            dfd,
   return TRUE;
 }
 
+/* Change the policy store location.
+ * Part of SELinux in Fedora >= 24: https://bugzilla.redhat.com/show_bug.cgi?id=1290659
+ */
+gboolean
+rpmostree_rootfs_prepare_selinux (int rootfs_dfd,
```

Comment:
Can we name this something more explicit, e.g. `rpmostree_set_selinux_etc_store_root` ? Or maybe `rpmostree_move_selinux_store` ?